simp_le.py

```diff
@@ -56,7 +56,7 @@
 VERSION = '0'
 URL = 'https://github.com/kuba/simp_le'
 
-LE_STAGING_URI = 'https://acme-staging.api.letsencrypt.org/directory'
+LE_STAGING_URI = 'https://acme-v01.api.letsencrypt.org/directory'
```

Comment:
Hmm, in my previous comment I explicitly said `create a new constant`, but after thinking about it for a while: `simp_le` is not a library - we don't need to store unnecessary/unused stuff here.

It's good as it is. Thanks.